iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerAdderImpl.java

```diff
@@ -75,17 +90,23 @@ protected void checkParams() {
             if (Double.isNaN(x)) {
                 throw new ValidationException(this, "x is not set");
             }
+            if (Double.isNaN(g)) {
+                throw new ValidationException(this, "g is not set");
+            }
+            if (Double.isNaN(b)) {
+                throw new ValidationException(this, "b is not set");
+            }
             if (Double.isNaN(ratedU)) {
                 throw new ValidationException(this, "rated u is not set");
             }
```

Comment:
Same remark as above, replace by:
`ValidationUtil.checkRatedU(this, ratedU, "");`

Reply:
Done